draft-ietf-mls-protocol.md

```diff
@@ -1188,6 +1191,32 @@ A step in this chain (the second subscript) is called a "generation".
 The details of application key derivation are described in the
 {{astree}} section below.
 
+## Exporters
+
+The main MLS key schedule provides an `exporter_secret` which can
+be used by an application as the basis to derive new secrets called
+`exported_value` outside the MLS layer.
+
+~~~~~
+exported_value =
+    HKDF-Expand-Label(exporter_secret, Label, Context, output_length)
```

Comment:
This is different from the [TLS construction](https://tools.ietf.org/html/rfc8446#section-7.5)

```
   TLS-Exporter(label, context_value, key_length) =
       HKDF-Expand-Label(Derive-Secret(Secret, label, ""),
                         "exporter", Hash(context_value), key_length)
```

Notice that the "Derive-Secret" is missing from the construction in the PR.  What's the reasoning here?

Reply:
In a similar vein, it might be useful to say something about labels, as the [TLS exporter RFC does](https://tools.ietf.org/html/rfc5705#section-4).  For example, do we want to have a registry?

Reply:
Fix pending... Any intuition about the registry story ?